people.md

```diff
@@ -34,6 +34,17 @@ postdoc:
       [<i class='fab fa-github-square fa-2x'></i>](https://github.com/yujulian0168)&nbsp;&nbsp;
       [<i class='fab fa-linkedin fa-2x'></i>](https://www.linkedin.com/in/julian-yu-38836a141/)&nbsp;&nbsp;
       [<i class='ai ai-cv-square ai-2x'></i>](assets/files/YU,J-CV2020.pdf)
+
+phd:
+  - title: Henry Secaira Morocho
+    image_path: assets/images/people/HenryPhoto.jpeg
```

Comment:
@HSecaira Don't forget to modify the image file name too since your cropped image has a different name.